user-docs/documentation/docs/how-to-guides/5-deploy-cdk-constructs.md

```diff
@@ -0,0 +1,220 @@
+---
+sidebar_position: 5
+---
+
+# Deploy CDK Constructs
+
+## Deploy a simple CDK construct
+
+Define your CDK construct and provide it in the `construct` key in your `serverless.ts` configuration file.
+
+:::note
+This is only compatible with constructs defined using the CDK V2.
+:::
+
+```typescript
+import { ServerlessCdkPluginConfig } from '@swarmion/serverless-cdk-plugin';
+
+class MyConstruct extends Construct {
```

Comment:
Explicit the import of `Construct`, we cannot copy paste the code as is

Reply:
Let's do it for all the imported variables

Reply:
And across all the examples

Reply:
Done !
Also added the AWS imports from `@types/serverless`